packages/plugin-core/package.json

```diff
@@ -795,6 +797,7 @@
     "mocha": "^7.1.2",
     "ts-loader": "^8.0.0",
     "ts-node": "^8.10.2",
+    "vsce": "^1.99.0",
```

Comment:
this should be the same version we use in `dendron-cli` so that `lerna` doesn't have two packages and for compatibility issues